backend/plugin/parser/cassandra/query_span_extractor.go

```diff
@@ -60,6 +64,13 @@ func (e *querySpanExtractor) EnterSelect_(ctx *cql.Select_Context) {
 	results := e.extractSelectElements(ctx.SelectElements(), keyspace, table)
 	e.querySpan.Results = results
 
+	// Add all result columns to SourceColumns (simplified implementation)
+	for _, result := range results {
+		for col := range result.SourceColumns {
+			e.querySpan.SourceColumns[col] = true
```

Comment:
`QuerySpan.SourceColumns` is used to store the table resource accessed by the query now.